types/test/errors.ts

```diff
@@ -5,7 +5,7 @@ import { OptimisticLockError } from '../lib/errors';
 
 async function test() {
   try {
-    await User.create({ username: 'john_doe' });
```

Comment:
I made `firstName` required in the attributes I defined, so I updated this code to match